app/controllers/concerns/op_turbo/component_stream.rb

```diff
@@ -83,6 +83,10 @@ def render_error_flash_message_via_turbo_stream(**)
       render_flash_message_via_turbo_stream(**, scheme: :danger, icon: :stop)
     end
 
+    def render_aria_update_message(message:, type:, role: "alert")
```

Comment:
It would be good to have a sensible default for `type`[^1] here, i.e.

```suggestion
    def render_aria_update_message(message:, type: :polite, role: :alert)
```

Also symbols are slightly more idiomatic Ruby[^2].

[^1]: see other comment about naming - `type` vs `politeness`.
[^2]: see other comment about `role` not being used the custom Turbo Stream Action.